src/SelfScreenerContext.spec.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/exhaustive-deps*/
```

Comment:
@johnschoeman This was because this rule was directing me to put dependencies in the array of the `useEffect` hook, but when I did that in the test env, it entered an infinite loop and crashed 😅. 

Since this is a test, and I wanted the hook to run only only once period, I thought it was okay to keep the dependency array empty, so I eslint-disabled so that it wouldn't give warning when we run yarn validate.

Could definitely be worth taking a second look, though! 🧐